tools/toolchain/scripts/install_libvdwxc.sh

```diff
@@ -0,0 +1,106 @@
+#!/bin/bash -e
+[ "${BASH_SOURCE[0]}" ] && SCRIPT_NAME="${BASH_SOURCE[0]}" || SCRIPT_NAME=$0
+SCRIPT_DIR="$(cd "$(dirname "$SCRIPT_NAME")" && pwd -P)"
+libvdwxc_ver=${libvdwxc_ver:-master}
+
+source "${SCRIPT_DIR}"/common_vars.sh
+source "${SCRIPT_DIR}"/tool_kit.sh
+source "${SCRIPT_DIR}"/signal_trap.sh
+
+with_libvdwxc=${1:-__INSTALL__}
+
+[ -f "${BUILDDIR}/setup_libvdwxc" ] && rm "${BUILDDIR}/setup_libvdwxc"
+
+! [ -d "${BUILDDIR}" ] && mkdir -p "${BUILDDIR}"
+cd "${BUILDDIR}"
+case "$with_libvdwxc" in
+    __INSTALL__)
+
+        require_env MPI_LDFLAGS
+        require_env MPI_LIBS
+        require_env FFTW_LDFLAGS
+        require_env FFTW_LIBS
+        require_env FFTW_CFLAGS
+
+        echo "==================== Installing libvdwxc ===================="
+        pkg_install_dir="${INSTALLDIR}/libvdwxc-${libvdwxc_ver}"
+        install_lock_file="$pkg_install_dir/install_successful"
+        if verify_checksums "${install_lock_file}" ; then
+            echo "libvdwxc-${libvdwxc_ver} is already installed, skipping it."
+        else
+            if [ -f libvdwxc-${libvdwxc_ver}.tar.gz ] ; then
+                echo "libvdwxc-${libvdwxc_ver}.tar.gz is found"
+            else
+                # do not remove this. They do not publish official version often
+                wget "https://gitlab.com/libvdwxc/libvdwxc/repository/archive.tar.gz?ref=master" -O libvdwxc-master.tar.gz
```

Comment:
I uploaded today's version to here: https://www.cp2k.org/static/downloads/libvdwxc-master-429a80027a2ec2c97e2f6d9a3dc84843f2739865.tar.gz . Please use that URL to make the build reproducible.